tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "include": ["src"],
+    "include": ["src", "test"],
```

Comment:
Is there any side effects to including `test` over here?

Reply:
Tests files now will be typechecked. 
The build won't be affected as the entry point for `tsup` (`src/index.ts` by default) and the build graph does not reference `/tests` code.